docs/source/en/preprocessing.mdx

```diff
@@ -197,23 +199,25 @@ array([[1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0],
 </tf>
 </frameworkcontent>
 
-## Audio
+## Feature extractor
```

Comment:
I'd leave Audio and Vision as level 2 header (especially since feature extractor will soon not be the right class for vision).